packages/hint-compat-api/src/html.ts

```diff
@@ -74,6 +74,7 @@ export default class HTMLCompatHint implements IHint {
 
     public constructor(context: HintContext) {
         const ignore = resolveIgnore([
+            'autocomplete',
```

Comment:
Slight tweak since we're trying to ignore the `autocomplete` attribute on input elements (still omitting the element name since this should also apply to `<textarea>` and `<select>` so we might as well be generic):
```suggestion
            '[autocomplete]',
```

Reply:
Actually ignore my suggestion. I just double-checked the code and realized we didn't use selector syntax for this when it's just a bare attribute name 🤦‍♂️ 

Your original change is correct, but I'd still like to add a test case so we're validating it.